### ISSUE TITLE
KeyError: 'CDA:ObIpo'

### ISSUE THREAD

**Issue by Andrettin** [jkersey/BlenderSpriter#1]
I have followed the instructions and managed to generate a spritesheet for player2.blend. But when using another model I get the following error:

File "<string>", line 79, in render
KeyError: 'CDA:ObIpo'

Any ideas on what may be causing this?

**Comment by jkersey** [comment]
No, haven't seen that before. Line 79 is trying to iterate through a list of animations it found in the .blend file. Maybe try adding CDA:ObIpo to the list of ignored_actions in config.ini? (also, thanks for checking this project out!)

**Comment by Andrettin** [comment]
Thanks for the response! Unfortunately, that didn't solve the issue. Nor did renaming that animation to something else. Unlinking it allowed the script to go further than that, but then I got another error (in line 62), saying that the system can't find "tmp\\*.*"

For reference, this is the model I'm trying to generate a spritesheet for:
http://opengameart.org/content/flare-mod-peddler-goblin

**Comment by jkersey** [comment]
Interesting, the CDA:ObIpo is the staff, not sure why it's not happy with that. I'll dig in to it in a few hours after work.

**Comment by jkersey** [comment]
Hey Andrettin, thanks for your patience!

If a tmp directory isn't being created, you'll need to make one in the current directory. You may need one for output/images as well.

These were the changes I had to make to get it to work, and I completely forgot about the animation_config directory when I was putting the description together. That file controls what animations are read from the .blend file as well as the directions the player should face.

I attached the files I edited, I'm going to modify the script to hopefully accommodate for these issues as soon as I can get my editor configured again.

#### In the .blend file: ####
Change the output resolution 64px x 64px, change the 25%

 below that to 100%
in the **Outliner**, open the **RenderPlatform**, grab the icon next to **GobArm** and drag it to the left until it offers to clear the parent
Rename **GobArm** to **Armature**
Under the new **Armature** is **Animation**, and beneath that is **Action**. Rename **Action** to **all**
At the top of the outline is **Scene**, rename that to **Player**

#### in animation_config/player.cfg: ####
rename **stand** to **all**, remove the other lines

#### in config.ini ####
add CDA:ObIpo to the list of ignored actions

[config.ini.zip](https://github.com/jkersey/BlenderSpriter/files/52759/config.ini.zip)
[animation_config.zip](https://github.com/jkersey/BlenderSpriter/files/52757/animation_config.zip)
[peddler.blend.zip](https://github.com/jkersey/BlenderSpriter/files/52758/peddler.blend.zip)

**Comment by Andrettin** [comment]
Hello jkersey,

First of all, I'm sorry for the delay in responding.

Thanks very much for your help! I tried generating spritesheets using the files you've attached; the previous error is gone, but unfortunately now I'm getting new ones:

"Error: File written by newer Blender binary (270.5), expect a loss of data!"
"Error, region type missing in - name: "Image", id:6"

**Comment by jkersey** [comment]
wow, yeah, that's pretty weird, what version of Blender and what platform are you using?

**Comment by Andrettin** [comment]
I'm on Windows 8, and am using Blender 2.68a (since that's what you used for the description of how to set up the script).

**Comment by jkersey** [comment]
Ah, I'm betting the goblin model was made with a newer Blender than 2.68. Totally safe to use the latest Blender with the script.
